### PR TITLE
Consolidate gitignore

### PR DESCRIPTION
**Pull request by aflag**
Closes: https://github.com/ScottishCovidResponse/SCRCIssueTracking/issues/569

### CODE REVIEW

**Comment by anthonyohare** [comment]
Happy with this change.

File: .gitignore [hunk 1]
```diff
@@ -10,20 +10,20 @@ __pycache__/
...
 .DS_Store
+
+# Ignore data output regardless of format
 tests/test_data/data_pipeline_inputs/output/**
 sample_input_files/output/*
 access*yaml
 
-# Do not ignore sample inputs
+# Do not ignore the csv files inside the sample and test data directories. The
+# sample_input_files directory contains yamls that will not be ignored because
```
Comment (bobturneruk) [inline .gitignore:21]: Can you rephrase this, please? So many negatives! I don't follow!
